elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/query/ColumnProjection.java

```diff
@@ -32,6 +33,16 @@ default String getAlias() {
         return getName();
     }
 
+    default String getSQLAlias() {
```

Comment:
We tried to keep the concept of SQL out of the query engine classes like ColumnProjection.  This is why I thought the column ID might work better here.

Reply:
renamed to getSafeAlias method